pkgs/applications/networking/remote/x2goserver/default.nix

```diff
@@ -0,0 +1,66 @@
+{ stdenv, lib, fetchurl, makeWrapper, perl, perlPackages, which, nx-libs
+, utillinux, coreutils, glibc, gawk, gnused, gnugrep, findutils, xorg
+, nettools, iproute, bc, procps, psmisc, lsof, pwgen, openssh, sshfs
+}:
+
+let binaryDeps = [
+  perl which nx-libs utillinux coreutils glibc.bin gawk gnused gnugrep
+  findutils nettools iproute bc procps psmisc lsof pwgen openssh sshfs
+  xorg.xauth xorg.xinit xorg.xrandr xorg.xmodmap xorg.xwininfo xorg.fontutil
+  xorg.xkbcomp xorg.setxkbmap
+]; in
+stdenv.mkDerivation rec {
+  name = "x2goserver-${version}";
```

Comment:
Please replace with `pname = "x2goserver";` and then you won't need to explicitly set `name` as it will be taken care of for you.

Reply:
Fixed.